shared-bindings/busio/UART.c

```diff
@@ -55,7 +55,7 @@
 //|         :param ~microcontroller.Pin rs485_dir: the output pin for rs485 direction setting, or ``None`` if rs485 not in use.
 //|         :param bool rs485_invert: rs485_dir pin active high when set. Active low otherwise.
 //|         :param int baudrate: the transmit and receive speed.
-//|         :param int bits:  the number of bits per byte, 7, 8 or 9.
+//|         :param int bits:  the number of bits per byte, 5 to 8.
```

Comment:
Let's not change this now. 8 bits is by far the most common.

This drops any existing 9 bit support and existing ports would have to check for 5 and 6 now too.

Reply:
I did check with all ports... 9 bits is only supported by stm.
I'll extend the range to 9 bits... as for checks... only i.mx requires a change, nrf doesn't check for bits... I'll add the check.

Port | Bits
--- | ---
samd | 5-8
cxd56 | 8
esp32s2 | 5-8
litex | N/A
i.mx | 7, 8
nrf | 8
rp2 | 5-8
stm | 8, 9

Reply:
Thank you!